src/components/Card/README.md

```diff
@@ -460,6 +460,89 @@ Use to render custom content such as icons, links, or buttons in a card section'
 </Card>
 ```
 
+### Card with all of its elements
+
+<!-- example-for: web -->
+
+Use as a broad example that includes most props available to card.
+
+```jsx
+<Card
+  secondaryFooterAction={{content: 'Dismiss'}}
+  primaryFooterAction={{content: 'Export Report'}}
+>
+  <Card.Header
+    actions={[
+      {
+        content: 'Total Sales',
+      },
+    ]}
+    title="Sales"
+  >
+    <Popover
+      active={false}
+      activator={
+        <Button disclosure plain>
+          View Sales
+        </Button>
+      }
+      onClose={() => {}}
+    >
+      <ActionList items={[{content: 'Gross Sales'}, {content: 'Net Sales'}]} />
+    </Popover>
+  </Card.Header>
+  <Card.Section>
+    <TextContainer>
+      You can use sales reports to see information about your customers' orders
+      based on criteria such as sales over time, by channel, or by staff.
+    </TextContainer>
+  </Card.Section>
+  <Card.Section title="Total Sales Breakdown">
+    <ResourceList
+      resourceName={{singular: 'sale', plural: 'sales'}}
+      items={[
+        {
+          sales: 'Orders',
+          amount: 'USD$0.00',
+          url: 'reports/orders',
+        },
+        {
+          sales: 'Returns',
+          amount: '-USD$250.00',
```

Comment:
Where did this piece come from? Is that how we display a debt or amount owing? I would have expected `USD-$250.00`

Reply:
I got it from the admin dashboard:

![](https://screenshot.click/11-10-2g5gh-3whdd.jpg)

Reply:
Thanks. It's correct, just not what I would have expected.

Reply:
I wonder what the decision behind it is. My assumption would be to reflect the urgency.